google-login-plugin/src/com/google/gct/login/stats/UsageTrackerService.java

```diff
@@ -48,11 +49,12 @@ public static UsageTracker getInstance() {
    *
    */
   static UsageTracker getInstance(@Nullable String key) {
-    UsageTracker instance = (key == null) ? null : COLLECTOR.findSingle(key);
-    if (instance != null) {
-      return instance;
+    if (usageTrackerManager.isTrackingEnabled()) {
```

Comment:
There is an issue with putting this check here, enabling tracking via dialog after a tracking event happens doesn't change the tracker to the google tracker.  They have to enable tracking, then restart the IDE.  I believe the intellij/studio guys do this check in their tracker.